### PR TITLE
docs: few minor fixes.

### PR DESCRIPTION
**Pull request by moonlit-melody**
Hi! I hope you are having a great day.
I am submitting a pull request with the following changes:

1. __Updated explanation to align with code and fix a minor grammar mistake in the documentation related to C Bit Fields.__ [1cfca66](https://github.com/MicrosoftDocs/cpp-docs/pull/4653/commits/1cfca6679900fe0c3a07fd390dfd0a3ba7544386)
2. __Clarify the description regarding the default size in the documentation related to Storage and Alignment of Structures.__ [532242a](https://github.com/MicrosoftDocs/cpp-docs/pull/4653/commits/532242a39f2c9cf70a2ff4cf755c821ce9947f6f)
3. __Fix a comment formatting error in the documentation related to Union Declarations.__ [f92be36](https://github.com/MicrosoftDocs/cpp-docs/pull/4653/commits/f92be3668f8cf0d9648c250f5b7f0ea560338b34)

Thank you for your time and consideration in reviewing these changes.
Best regards,
melody

### CODE REVIEW

File: docs/c-language/c-bit-fields.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The *`constant-expression`* specifies the width of the field in bits. The *`type
 
 Unnamed bit fields can't be referenced, and their contents at run time are unpredictable. They can be used as "dummy" fields, for alignment purposes. An unnamed bit field whose width is specified as 0 guarantees that storage for the member following it in the *struct-declaration-list* begins on an **`int`** boundary.
 
-Bit fields must also be long enough to contain the bit pattern. For example, these two statements aren't legal:
+Bit fields should not exceed the total number of bits of their underlying type. For example, these two statements aren't legal:
```
Comment (TylerMSFT) [inline docs/c-language/c-bit-fields.md:22]: Maybe the clearest way to say this is: The number of bits in a bit field must be less than or equal to the size of the underlying type.

**Review by TylerMSFT — CHANGES_REQUESTED** [review]
Thanks, @moonlit-melody! I think we should tweak the description of how many bits can be in the bit field.

File: docs/c-language/c-bit-fields.md (hunk 1 above)
Comment (moonlit-melody) [reply docs/c-language/c-bit-fields.md]: Appreciate your input, Tyler! The updated explanation now matches the code correctly.

**Review by TylerMSFT — APPROVED** [review]
Thanks, @moonlit-melody!

**Comment by TylerMSFT** [comment]
#sign-off

**Pull request merged**